samples/monitoring_schedule/README.md

```diff
@@ -4,7 +4,7 @@ This sample demonstrates how to start Monitoring scheduling using your own Monit
 
 ## Prerequisites
 
-This sample assumes that you have already configured an Kubernetes cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do). It also assumes you have a job definition of one of these [types](/samples/job_definitions).
+This sample assumes that you have already configured an Kubernetes cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](/README.md#getting-started). It also assumes you have a job definition of one of these [types](/samples/job_definitions).
```

Comment:
minor nit: ~~It also assumes~~ you have a job definition
rephrase sentence, sounds repetitive

Reply:
rephrased